### PR TITLE
feat: get route

### PR DESCRIPTION
**Pull request by tranmarta**

### CODE REVIEW

**Review by NerdbordMentor — CHANGES_REQUESTED** [review]
## Code Review Report

### Task Goal
The task was to implement an endpoint for fetching JSON data.

### Requirements
- `trains.json` should be fetched as a data source.
- Required endpoint schema: `GET /trains`.

### Code Changes Review

The provided code changes do not cover the task requirements. Here are the issues:

1. The code does not define an endpoint `GET /trains`. The code should have been written inside an Express.js route handler like this:
```javascript
app.get('/trains', (req, res) => {
    // Fetching and response logic here
});
```

2. The `fetch` function is used incorrectly. It seems like it's trying to fetch a file called `trains.json`, but the filename is not enclosed in quotes, which will cause a syntax error. Also, `fetch` is a browser-based API and it's not available in Node.js environment by default. To read a JSON file in Node.js, you can use the `fs` module.

3. The data from `trains.json` is hardcoded into the fetch promise resolution. This is not fetching data from the `trains.json` file.

4. The fetched data is not sent as a response to the client. The Express.js `res` object should be used to send the data back to the client.

5. There is no error handling. If fetching the data fails for any reason, the server will crash because there are no `.catch` blocks to handle potential errors.

In conclusion, the provided code changes do not meet the task requirements. The developer needs to define the required endpoint, fetch data from the `trains.json` file correctly, and send this data as a response to the client.
